config.json

```diff
@@ -201,6 +201,21 @@
       ]
     },
     {
+      "core": false,
+      "difficulty": 4,
+      "slug": "binary-search",
+      "topics": [
+        "arrays",
+        "control_flow_if_else_statements",
+        "control_flow_loops",
+        "searching"
+      ],
+      "unlocked_by": null,
+      "uuid": "abdd7aa3-25f6-4d27-a4dc-6bd1ea2f718c"
+    },
+    {
+      "core": false,
```

Comment:
There's a merge error here where you're adding to the next problem as well. You also need to re-format your settings.

The isogram merge had the same issue, and I missed it too. (I fixed that already)

Reply:
Done. Thanks for the pointer.